src/components/shared/Exercises/ExerComplete.tsx

```diff
@@ -0,0 +1,20 @@
+import '../../../styles/Exercises/ExerComplete.scss';
+import encouragingTurtle from '../../../assets/EncouragingTurtle-Solo.svg';
+
+interface ExerCompleteInputs {
+  exerciseName: string;
+}
+
+function ExerComplete({ exerciseName }: ExerCompleteInputs): JSX.Element {
```

Comment:
This is a nitpick, but let's rename this to ExerciseComplete, we haven't abbreviated exercise elsewhere and it makes for cleaner code